### PR TITLE
Update paper.md+biobliography

### PR DESCRIPTION
**Pull request by greatfireball**

### CODE REVIEW

File: paper/paper.bib [hunk 1]
```diff
@@ -1,13 +1,178 @@
...
+  Doi                      = {10.1007/978-3-642-37195-0_13},
+  ISBN                     = {978-3-642-37195-0},
+  Url                      = {https://doi.org/10.1007/978-3-642-37195-0_13}
+}
+
+@Article{sra2008,
+  Title                    = {Database resources of the National Center for Biotechnology Information},
+  Author                   = {Wheeler, David L. and Barrett, Tanya and Benson, Dennis A. and Bryant, Stephen H. and Canese, Kathi and Chetvernin, Vyacheslav and Church, Deanna M. and DiCuccio, Michael and Edgar, Ron and Federhen, Scott and Feolo, Michael and Geer, Lewis Y. and Helmberg, Wolfgang and Kapustin, Yuri and Khovayko, Oleg and Landsman, David and Lipman, David J. and Madden, Thomas L. and Maglott, Donna R. and Miller, Vadim and Ostell, James and Pruitt, Kim D. and Schuler, Gregory D. and Shumway, Martin and Sequeira, Edwin and Sherry, Steven T. and Sirotkin, Karl and Souvorov, Alexandre and Starchenko, Grigory and Tatusov, Roman L. and Tatusova, Tatiana A. and Wagner, Lukas and Yaschenko, Eugene},
+  Journal                  = {Nucleic Acids Research},
+  Year                     = {2008},
```
Comment (iimog) [inline paper/paper.bib]: Why exactly the publication of 2008? A similar article is published every year, most recently https://doi.org/10.1093/nar/gkw1071 
Or is there a specific reason I am missing?

File: paper/paper.md [hunk 2]
```diff
@@ -39,29 +39,28 @@ affiliations:
...
+Using our demo set, one can achieve a single contig assembly of the chloroplast of *Spinacia oleracea* .
+The final chloroplast sequence can be further annotated with tools like DOGMA [@dogma2004], cpGAVAS [@Liu2012] and VERDANT [@Mckain2017].
+Such assemblies, can be used to remove chloroplast reads before a genomic assembly of the remaining nuclear DNA.
+Moreover, chloroplast genomes are useful in phylogenetic reconstruction [@huang_2016] or barcoding applications [@Coissac_2016].
+A similar tool, aiming the assembly of whole chloroplast genomes is the Python program org.ASM [https://git.metabarcoding.org/org-asm/org-asm], but it is not production ready, yet.
+Also plasmid SPAdes [@plasmidspades_2016] could possibly be used for this purpose although it is not intended for it.
+In the future, we plan to use our chloroExtractor to screen NCBI's Sequence Read Archive [@sra2008] for chloroplast genomes in public sequencing datasets that are not yet available in chloroplast databases, eg. chloroDB [@chlordb2006] to broaden our knowledge about chloroplasts.
```
Comment (iimog) [inline paper/paper.md]: This reference is to NCBI as a whole. That is not a bad idea, but maybe we should add the more specific citation for SRA: https://doi.org/10.1093/nar/gkq1019

File: paper/paper.bib (hunk 1 above)
Comment (greatfireball) [reply paper/paper.bib]: wilco

File: paper/paper.md (hunk 2 above)
Comment (greatfireball) [reply paper/paper.md]: okay... Will add that, but we will remove the citation in paper.bib with your issue above?

**Comment by greatfireball** [comment]
Was sagst du zu den Änderungen @iimog

**Pull request merged**